README.md

```diff
@@ -951,10 +971,24 @@ They can be overridden for testing purposes in [the same way as other settings](
 
 ##### AWS-specific settings
 
-AWS-specific settings are automatically set based on calls to the Instance MetaData Service (IMDS).
+* `settings.aws.config`: The base64 encoded content to use for AWS configuration (e.g. `base64 ~/.aws/config`).
```

Comment:
Should consider mentioning that these settings are not necessary when relying on the instance role (either here or below where you have a `**Note**`)?

Reply:
Good call, added a note.